### DISCUSSION TITLE
Superslicer will not slice correctly

### DISCUSSION THREAD

**Discussion by TomasL-SE** [supermerill/SuperSlicer#3886] [Q&A]
I have an model that superslicer does not slice correctly.
This is how it is supposed to look like
![as33000](https://github.com/supermerill/SuperSlicer/assets/121312514/527cf628-6ce4-460a-811c-56fb798980f7)
after slicing
![as33001](https://github.com/supermerill/SuperSlicer/assets/121312514/31ff5918-596c-48f6-aded-5c6d59008f01)
and the model
[asr-33-tinker.zip](https://github.com/supermerill/SuperSlicer/files/11919539/asr-33-tinker.zip)
Prusaslicer is somewhat better to do it, however the print results are pure crap

**Comment by neophyl** [comment]
The features on that model at the size it is are too small for your extrusion width settings (with classic slicing).  Prusa Slicer uses arachne by by default which will try and print smaller details, although often with limited sucess.

Using 2.5.59.2 and also selecting arachne as the slicing engine I can get it to slice like this with my normal 0.2mm layer profile.
![image](https://github.com/supermerill/SuperSlicer/assets/46955527/173599f9-c5b1-4c65-93d0-80b83393ac6a)

But I doubt it will actually print any better than PS.  For models that size and with that level of detail you need to use a smaller nozzle to allow much smaller details to be printed.  Or ideally use a resin printer as they are designed for small detailed things.

**Reply by TomasL-SE** [reply]
Well, apparently there are people that has printed it with success, sliced with prusa.
This is what I got from the creator
![Screenshot_20230630_150848_Gmail](https://github.com/supermerill/SuperSlicer/assets/121312514/dec77215-e8d0-47fe-b462-280ec9e4dc72)

**Comment by neophyl** [comment]
Yeah note they are using a 0.05 layer height.  I did point out that I was using 0.2.   They also haven't shown what arachne settings they are using so all we can do is assume prusa defaults.   OR their extrusion width values etc.  Just ask them to save the 3mf project file they used in PS and send it to you.  The 3mf will then contain all the settings and that way you can compare and actually see their slice preview and examine it in detail
Heres one sliced at 0.05 the same as they used.
![image](https://github.com/supermerill/SuperSlicer/assets/46955527/b3dade91-3f4b-4072-a8cd-84a976028828)


BTW They cant have printed that slice as pictured.  Note the skirt only goes around the small protrusion on the bottom, they would have needed to add support to actually make it printable at all as most of the 'bottom' layers are actually several layers up off the bed.  Have they actually shown you a picture of an actual print ?  

Besides if PS is doing a better job then use PS.
